packages/orchestration/src/orchestration-types.ts

```diff
@@ -63,6 +63,22 @@ export interface OrchestrationModuleConfig {
   llm: LlmModuleConfig;
   /**
    * Filtering module configuration.
+   * @example
+   * Configuring an input and output filter:
```

Comment:
[pp] I feel this line is not need (it just repeats what is visible from the code) without giving more info